### DISCUSSION TITLE
R2S下载哪个固件啊

### DISCUSSION THREAD

**Discussion by wwjj999** [haiibo/OpenWrt#133] [Q&A]
R2S下载哪个固件啊，请教了

**Comment by ZacharyBacon** [comment]
[openwrt-rockchip-armv8-friendlyarm_nanopi-r2s-squashfs-sysupgrade.img.gz](https://github.com/haiibo/OpenWrt/releases/download/Rockchip/openwrt-rockchip-armv8-friendlyarm_nanopi-r2s-squashfs-sysupgrade.img.gz)
